samples/bluetooth/peripheral_esp/src/main.c

```diff
@@ -32,66 +32,66 @@
-#define ESS_COUNT 				0x07
+#define ESS_DESC_SAMPLING_UNSPECIFIED				0x00
+#define ESS_DESC_SAMPLING_INSTANTANEOUS				0x01
+#define ESS_DESC_SAMPLING_ARITHMETIC_MEAN			0x02
+#define ESS_DESC_SAMPLING_RMS					0x03
+#define ESS_DESC_SAMPLING_MAXIMUM				0x04
+#define ESS_DESC_SAMPLING_MINIMUM				0x05
+#define ESS_DESC_SAMPLING_ACCUMULATED				0x06
+#define ESS_DESC_SAMPLING_COUNT					0x07
 
 /* ES Measurement Descriptor - Applications */
-#define ESS_UNSPECIFIED 				0x00
-#define ESS_AIR 					0x01
-#define ESS_WATER 					0x02
-#define ESS_BAROMETRIC 					0x03
-#define ESS_SOIL 					0x04
-#define ESS_INFRARED 					0x05
-#define ESS_MAP_DATABASE 				0x06
-#define ESS_BAROMETRIC_ELEVATION_SOURCE 		0x07
-#define ESS_GPS_ONLY_ELEVATION_SOURCE 			0x08
-#define ESS_GPS_AND_MAP_DATABASE_ELEVATION_SOURCE 	0x09
-#define ESS_VERTICAL_DATUM_ELEVATION_SOURCE 		0x0A
-#define ESS_ONSHORE 					0x0B
-#define ESS_ONBOARD_VESSEL_OR_VEHICLE 			0x0C
-#define ESS_FRONT 					0x0D
-#define ESS_BACK_REAR 					0x0E
-#define ESS_UPPER 					0x0F
-#define ESS_LOWER 					0x10
-#define ESS_PRIMARY 					0x11
-#define ESS_SECONDARY 					0x12
-#define ESS_OUTDOOR 					0x13
-#define ESS_INDOOR 					0x14
-#define ESS_TOP 					0x15
-#define ESS_BOTTOM 					0x16
-#define ESS_MAIN 					0x17
-#define ESS_BACKUP 					0x18
-#define ESS_AUXILIARY 					0x19
-#define ESS_SUPPLEMENTARY 				0x1A
-#define ESS_INSIDE 					0x1B
-#define ESS_OUTSIDE 					0x1C
-#define ESS_LEFT 					0x1D
-#define ESS_RIGHT 					0x1E
-#define ESS_INTERNAL 					0x1F
-#define ESS_EXTERNAL 					0x20
-#define ESS_SOLAR 					0x21
+#define ESS_DESC_APP_UNSPECIFIED				0x00
+#define ESS_DESC_APP_AIR					0x01
+#define ESS_DESC_APP_WATER					0x02
+#define ESS_DESC_APP_BAROMETRIC					0x03
+#define ESS_DESC_APP_SOIL					0x04
+#define ESS_DESC_APP_INFRARED					0x05
+#define ESS_DESC_APP_MAP_DATABASE				0x06
+#define ESS_DESC_APP_BAROMETRIC_ELEVATION_SOURCE		0x07
+#define ESS_DESC_APP_GPS_ONLY_ELEVATION_SOURCE			0x08
+#define ESS_DESC_APP_GPS_AND_MAP_DATABASE_ELEVATION_SOURCE	0x09
+#define ESS_DESC_APP_VERTICAL_DATUM_ELEVATION_SOURCE		0x0A
+#define ESS_DESC_APP_ONSHORE					0x0B
+#define ESS_DESC_APP_ONBOARD_VESSEL_OR_VEHICLE			0x0C
+#define ESS_DESC_APP_FRONT					0x0D
+#define ESS_DESC_APP_BACK_REAR					0x0E
+#define ESS_DESC_APP_UPPER					0x0F
+#define ESS_DESC_APP_LOWER					0x10
+#define ESS_DESC_APP_PRIMARY					0x11
+#define ESS_DESC_APP_SECONDARY					0x12
+#define ESS_DESC_APP_OUTDOOR					0x13
+#define ESS_DESC_APP_INDOOR					0x14
+#define ESS_DESC_APP_TOP					0x15
+#define ESS_DESC_APP_BOTTOM					0x16
+#define ESS_DESC_APP_MAIN					0x17
+#define ESS_DESC_APP_BACKUP					0x18
+#define ESS_DESC_APP_AUXILIARY					0x19
+#define ESS_DESC_APP_SUPPLEMENTARY				0x1A
+#define ESS_DESC_APP_INSIDE					0x1B
+#define ESS_DESC_APP_OUTSIDE					0x1C
+#define ESS_DESC_APP_LEFT					0x1D
+#define ESS_DESC_APP_RIGHT					0x1E
+#define ESS_DESC_APP_INTERNAL					0x1F
+#define ESS_DESC_APP_EXTERNAL					0x20
+#define ESS_DESC_APP_SOLAR					0x21
```

Comment:
Thanks for also updating the other ones :)

Reply:
👍 I fixed the commit message and sign off so I think the PR compliance check should pass this time.

Reply:
Your commit author and sign off entries should be "Graham Lastname <email>", you are missing the last name right now

Reply:
I amended the commit and forced the author signature to get updated. I think that should work.